### DISCUSSION TITLE
How to show linkhints for non-link elements and click on the selected ones?

### DISCUSSION THREAD

**Discussion by MatejBransky** [gdh1995/vimium-c#960] [Q&A] [answered]
Hi, we have websites with bad a11y and sometimes the default `f` doesn't find links/buttons. In such a case I've found that I can use `yv` which uses `LinkHints.activateSelect` and shows me hints even for non-link elements but then I have to escape the visual mode and then I have to call `LinkHints.click`. How can I merge these steps into one?

**Comment by gdh1995** [comment, marked as answer]
Please use the `clickable` parameter instead - see https://github.com/gdh1995/vimium-c/wiki/Add-or-remove-hints

The idea of `to select and then click` may work but it's too complex to build a successful and universal key mapping - at least if a user is not familar with Vimium C source code.
